src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -599,6 +599,7 @@ engraving::ClefType Convert::clefFromMEI(const libmei::Clef& meiClef, bool& warn
         case (3): return engraving::ClefType::C3;
         case (4): return engraving::ClefType::C4;
         case (5): return engraving::ClefType::C5;
+        //case (6): return engraving::ClefType::C4_8VB;
```

Comment:
My best guess would be this:
```suggestion
        case (4):
            if (meiClef.GetDisPlace() == libmei::STAFFREL_basic_below && meiClef.GetDis() == libmei::OCTAVE_DIS_8) {
                return engraving::ClefType::C4_8VB;
            }
            return engraving::ClefType::C4;
        case (5): return engraving::ClefType::C5;
```

Reply:
Done

Reply:
unless @lpugin has concerns here...

Reply:
Looks good. `Convert::clefToMEI` should be adjusted accordingly.

Reply:
Ah, missed that. Will do Sunday or Monday

Reply:
Done